changelog.yaml

```diff
@@ -134,6 +134,13 @@
     added:
       - Updated `README.md` and its badges.
   date: 2023-09-20 17:00:00
+<<<<<<< HEAD
```

Comment:
@finpaul
- Get rid of git merge conflict metadata
- Instead of "Adds Montana tax logic." say "Updates Montana tax logic"
- Update the date and time stamp